celesta-java/src/main/java/ru/curs/celesta/Celesta.java

```diff
@@ -79,4 +89,46 @@ JSessionContext sessionContext(String userId, String sessionId) {
         return new JSessionContext(userId, sessionId);
     }
 
+    public Object runProc(String sesId, String qualifier, Object... args) {
+        JSessionContext sessionContext = this.sessions.get(sesId);
```

Comment:
I think this logic (get--check for null--throw Exception if null) is duplicated in JythonCelesta. Make a method in parent class and inherit?

Reply:
This logic is executed in different places of runProc/runPython. 
The best we can do is add a new package private method getSessionContext(String sessionId) to AbstractCelesta and invoke it.